ui/imports/utils/Utils.qml

```diff
@@ -137,11 +137,22 @@ QtObject {
                 `<a href="${link}">${link}</a>`
     }
 
+    function getStyledLink(linkText, linkUrl, hoveredLink, textColor = Theme.palette.directColor1, linkColor = Theme.palette.primaryColor1) {
```

Comment:
Any chance of unifying with `getLinkStyle`? 😄 
The 2 differences I see are the underline and link text. It's a game of spotting the difference when someone looks into these 2 functions.

Reply:
Yeah, I'll have a look how and where it's used :)

Reply:
Later, we want to have it in 2.30 asap